torch/_inductor/ir.py

```diff
@@ -8145,6 +8292,11 @@ def shim(dtypes, values):
                 # Proxies are iterable, but some methods expect tuples/lists
                 return tuple(result[i] for i in range(len(value_proxy)))
 
+            def sort(self, dtypes, values, stable, descending):
+                result = self._inner.sort(dtypes, values, stable, descending)
+                # Proxies are iterable, but some methods expect tuples/lists
+                return tuple(result[i] for i in range(len(values)))
```

Comment:
Is this not the same as `tuple(result)` (assuming `len(values) == len(result)`... which I don't see why it's true at first sight)

Reply:
`result` is a proxy variable here, so it doesn't have a know length to iterate over.